third_party/rust/chromium_crates_io/Cargo.toml

```diff
@@ -26,6 +26,7 @@ adblock-cxx = { version = "1" }
 constellation-cxx = "0.1"
 challenge-bypass-ristretto-cxx = "1"
 orchard = { version = "0.7.0",  default-features = false }
+incrementalmerkletree = { version = "0.5.1", features = ["legacy-api"]}
```

Comment:
we should not be adding this directly to chromium_crates_io/Cargo.toml. It should go in a Cargo.toml file along with the relevant code